### PR TITLE
Topic/gitattributes

### PR DESCRIPTION
**Pull request by paulp**

### CODE REVIEW

**Comment by paulp** [comment]
Review by @jsuereth

**Comment by paulp** [comment]
Yeah, oversight.  I added that.  Renormalizing didn't change any of them though.

**Comment by gkossakowski** [comment]
This commit is huge but it only modifies `*.check` files so it's unlikely to cause nasty merge conflicts for people busy with getting 2.10.x out. @jsuereth: can you lgtm?

**Comment by jsuereth** [comment]
Let me validate on windows first, sorry.  That machine is dog slow.

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+*.js      text
+*.sbt     text
+*.scala   text
+*.sh      text
+*.txt     text
+*.xml     text
+*.check   text
+
+# Windows-specific files get windows endings
+*.bat     eol=crlf
```
Comment (retronym) [inline .gitattributes]: *.cmd

**Comment by paulp** [comment]
PLS REBUILD ALL

**Comment by paulp** [comment]
@jsuereth ping / bump

**Comment by paulp** [comment]
I rebased it on 2.10.x.  This thing obviously drifts pretty quickly so what say we merge it.

**Comment by xeno-by** [comment]
A smoke test shows that this works okishly on a windows machine. Ant works, bat files from pack work, partest.bat works as well. I'd say we merge

**Comment by jsuereth** [comment]
Sorry, forgot to check on ye ole windows,   looks OK from the peanut
gallery too.

**Pull request merged**